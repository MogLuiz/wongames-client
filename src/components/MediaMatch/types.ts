// Types
import { DefaultBreakpoints } from "styled-media-query"

export type breakpoint = keyof DefaultBreakpoints
