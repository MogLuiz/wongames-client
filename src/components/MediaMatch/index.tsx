// Packages
import React from "react"

// Styles
import * as S from "./styles"

const MediaMatch: React.FC = () => (
  // -------------------------------------------------
  // Render
  // -------------------------------------------------
  <S.Wrapper>
    <h1>MediaMatch</h1>
  </S.Wrapper>
)

export default MediaMatch
