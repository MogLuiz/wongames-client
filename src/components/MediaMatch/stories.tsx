// Packages
import { Story, Meta } from "@storybook/react/types-6-0"

// Components
import MediaMatch from "."

export default {
  title: "MediaMatch",
  component: MediaMatch
} as Meta

export const Default: Story = () => <MediaMatch />
