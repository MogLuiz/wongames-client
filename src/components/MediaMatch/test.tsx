// Packages
import { render, screen } from "@testing-library/react"

// Components
import MediaMatch from "."

describe("<MediaMatch />", () => {
  it("should render the heading", () => {
    const { container } = render(<MediaMatch />)

    expect(
      screen.getByRole("heading", { name: /MediaMatch/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
