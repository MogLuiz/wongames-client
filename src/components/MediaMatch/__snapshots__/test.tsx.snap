// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaMatch /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    MediaMatch
  </h1>
</main>
`;
